src/shared/components/ShareWithColleagues/ShareWithColleagues.helpers.tsx

```diff
@@ -0,0 +1,12 @@
+import { flatten, groupBy } from 'lodash';
+
+import { ShareUserInfo, ShareUserInfoRights } from './ShareWithColleagues.types';
+
+export const sortShareUsers = (users: ShareUserInfo[]) => {
+	const groupedUsers = groupBy(users, 'rights');
+	return flatten([groupedUsers.EIGENAAR, groupedUsers.BIJDRAGER, groupedUsers.KIJKER]);
```

Comment:
You can also destruct the arrays into the parent array.
```suggestion
	return [
		...groupedUsers[ShareUserInfoRights.OWNER],
		...groupedUsers[ShareUserInfoRights.CONTRIBUTOR],
		...groupedUsers[ShareUserInfoRights.VIEWER]
	];
```